code.js

```diff
@@ -394,10 +395,35 @@ function keyPressed(e) {
             button_number(e.key) 
         }   
     }
+    if (e.key) {
+        key_combination[e.code] = e.key;
+    }
 }
 
 // function to capture keyup events
 function keyReleased(e){
+    if (key_combination['ControlLeft'] && key_combination['KeyV']) {
+        navigator.clipboard.readText().then(text => {
+            box = document.getElementById("box");
+            var isNumber = isFinite(parseInt(text));
+            if (isNumber){
+                var copy_number = parseInt(text)
+                box.innerText = copy_number
+                numbers.push(copy_number)
+            }
+            else{
+                box.innerText = 0
+                numbers.push(0)
+            }
```

Comment:
This needs to be deleted. When trying to paste non-number, nothing should change on the screen